### PR TITLE
tmpfs can now be set in install script

### PR DESCRIPTION
**Pull request by ArneTR**
@dan-mm I added a functionality so that the system will be put on a `tmpfs` for the `/tmp` directory in the install script.

This parameter triggers an input dialog.

Could you please adapt tests and pipelines so that the new code will run and then merge it?

### CODE REVIEW

File: install_linux.sh [hunk 1]
```diff
@@ -50,6 +55,13 @@ if [[ -z "$db_pw" ]] ; then
     read -sp "Please enter the new password to be set for the PostgreSQL DB: " db_pw
 fi
 
+if [[ $ask_tmpfs == true ]] ; then
+    read -p "We strongly recommend mounting /tmp on a tmpfs. Do you want to do that? (y/N)" tmpfs
+    if [[ "$tmpfs" == "Y" || "$tmpfs" == "y" ]] ; then
+        sudo systemctl enable /usr/share/systemd/tmp.mount
```
Comment (ribalba) [inline install_linux.sh:61]: We need a check there for fedora. The command under fedora is
```
sudo systemctl unmask --now tmp.mount
```

Just enabling tmp.mount does not work as it is not called by itself but wanted by local-fs.target

**Review by ribalba — CHANGES_REQUESTED** [review]
Should we not also write a little check function to make sure that when installed tmp is tmpfs and then offer to enable it?

**Comment by ArneTR** [comment]
> Should we not also write a little check function to make sure that when installed tmp is tmpfs and then offer to enable it?

I would argue this is part of the guard clauses / sanity checks.

File: install_linux.sh (hunk 1 above)
Comment (ArneTR) [reply install_linux.sh:61]: cool, ty!

**Pull request merged**